guides/plugins/plugins/framework/flow/add-flow-builder-action.md

```diff
@@ -0,0 +1,163 @@
+# Add customer flow action
+
+## Overview
+
+In this guide, you'll learn how to create a custom flow trigger in Shopware. Triggers are used by the flow builder. This example will introduce a new custom trigger. The shop owner is then able to define what to do with the new trigger.
+
+## Prerequisites
+
+In order to add your own custom flow trigger for your plugin, you first need a plugin as base. Therefore, you can refer to the [Plugin Base Guide](../../plugin-base-guide.md).
+
+You also should be familiar with [Add customer event](../event/add-custom-event.md) to know how to create an event.
+
```

Comment:
Should we also link to the (to be written) flow builder concept, so people understand how it works in general?